src/templates/partials/exportInterface.hbs

```diff
@@ -26,7 +26,11 @@ export namespace {{{name}}} {
 	{{/if}}
 	export enum {{{name}}} {
 		{{#each enum}}
+		{{#equals name ''}}
+		'' = {{{value}}},
+		{{else}}
```

Comment:
Why is this special case needed at all?

Reply:
The enum being empty would result in ` = ''` otherwise which doesn't compile. We want the enum name to have the single quotes in the name only for this case due to it being empty. It is a special use case since in the other enum values we don't need quotes

Reply:
@jstasiak But yeah, maybe we don't want `'' = ''` so I think I'll copy your approach and change this to `__EMPTY__`, you raise valid points. Will fix 👍

Reply:
Fixed. This has been changed so the name of the enum becomes `__EMPTY__` which makes this special case on the template unnecessary.